force-app/main/default/classes/DuplicateController.cls

```diff
@@ -0,0 +1,72 @@
+public with sharing class DuplicateController {
+    @AuraEnabled(Cacheable = true)
+    public static List<SObject> findDuplicates(String idFromPage) {
+        Id recordId = Id.valueOf(idFromPage);
+        String objName = recordId.getSObjectType().getDescribe().getName();
+        List<DuplicateRecordSet> duplicateRecordSet = getRecordSetList(recordId);
+
+        List<DuplicateRecordItem> itemsList = [SELECT recordId FROM DuplicateRecordItem WHERE DuplicateRecordSetId =: duplicateRecordSet];
+
+        List<String> recordIds = new List<String>();
+        for(DuplicateRecordItem item : itemsList){
+            recordIds.add(item.recordId);
+        }
```

Comment:
To bym wrzucił do osobnej metody pod tytułem: 

"getPotencialDuplicateRecordIds()" i tam wrzucił całość tego ciała.